src/Components/Forms/AuthForms/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,108 @@
+'use client';
+import React, { useState, useEffect } from 'react';
+import styles from '@/styles/credential.module.scss';
+import { useRouter } from 'next/navigation';
+import { FormikHelpers, useFormik } from 'formik';
+//@ts-ignore
+import { Button, Input } from 'technogetic-iron-smart-ui';
+import Image from 'next/image';
+import { ResetPasswordSchema } from '@/utils/schema';
+import { resetPasswordService } from '@/services/authServices';
+import { toast } from 'react-toastify';
+import { ResetFormValues } from '@/types/formsTypes';
+
+export const ResetPasswordForm: React.FC = () => {
+  const [token, setToken] = useState<string>('');
+  const [error, setError] = useState<string>('');
+  const router = useRouter();
+
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
+      const url = new URL(window?.location?.href);
+      const extractedToken = url.searchParams.get('token')!;
+      setToken(extractedToken);
+    }
+  }, []);
+
+  const initialValues: ResetFormValues = {
```

Comment:
keep initailValues in one place in all files, in other files this variable in top, here inside component.

Reply:
done.